build.js

```diff
@@ -1,13 +1,29 @@
 var $ = require('shelljs')
 
-if (!$.which('docker')) {
-  $.echo(
-    'This template requires docker to work. Please install docker and try again.',
-  )
-  $.exit(1)
+// Emscripten command:
```

Comment:
I also took the opportunity to put this on multiple lines so it can be scanned more easily